felix/cmd/calico-bpf/commands/conntrack.go

```diff
@@ -253,20 +328,78 @@ func runClean(c *cobra.Command, _ []string) {
 	}
 }
 
+type conntrackCreateCmd struct {
+	*cobra.Command
+
+	Version string `docopt:"<version>"`
+
+	version int
+}
+
+func newConntrackCreateCmd() *cobra.Command {
+	cmd := &conntrackCreateCmd{
+		Command: &cobra.Command{
+			Use:   "create <version>",
```

Comment:
similarly, should we have [--version=<version>] ? and we would assume the newest be default?